community/mei-by-laws.md

```diff
@@ -73,41 +77,51 @@ The Board is self-organizing; that is, there will be no formal regulations on ho
+The Board holds regular, virtual or in-person meetings, e.g. approximately every three months. As schedules and funding permit, the Board meets in person at least once per year. For every meeting, an agenda shall be published online prior to the meeting and minutes of the meeting shall be provided in a timely manner following the meeting. Depending on the availability of funding, the members of the Board may be refunded travel costs for this meeting.
+
+### 5.3 Terms & Elections
 
-Board members may resign from their commitments at any time but are expected to give adequate notice. After accepting the resignation, the Board may decide on how to proceed concerning the newly vacant position. Options should include but are not limited to holding a special election, appointing a replacement, or leaving the position vacant.
+The elected members of the Board each are elected for a three year term. Terms on the Board are staggered so that three positions are open for election each year. Terms of office begin January 1 at 2 pm UTC.
 
-In case of a special election to fill a vacancy between regular elections the newly elected Board member will serve for the remainder of the term of the vacant position.
+Exception to annual elections and term lengths will be made for the initial election of the Board and in case a Board member resigns.
 
 #### 5.3.1 Eligibility
 
 In order to be available for election to the Board, candidates must hold Personal Membership in MEI. Individuals who have previously served on the Board will be eligible for re-appointment.
 
 #### 5.3.2 Election procedure
 
-In order to determine a range of candidates, a four-week nomination phase will precede the elections, in which self-nominations are permitted. Individuals will be informed of their nomination and asked to confirm their willingness to serve on the Board determined. Elections will be held in a timely manner after determination of a final list of candidates. A ranked voting system without minimum voter turnout will be applied.
+In order to determine a range of candidates, a two-week nomination phase will precede the elections, in which self-nominations are permitted. Individuals will be informed of their nomination and asked to confirm their willingness to serve on the Board determined. Elections will be held in a timely manner after determination of a final list of candidates. Elections shall be open for at least two weeks. Elections shall close before the start of the terms of office of the elected candidates.
+All open board seats shall be contested by all candidates in one election. There is no minimum voter turnout requirement. Ballots shall allow ranked voting. The winners shall be determined by a counting method suitable for electing a group of candidates, e.g. the Single Transferable Vote (STV) method.
```

Comment:
Based on past experience in running elections: If there is a desire to be vague about the specific method chosen (to allow for this to change in the future without changing the by-laws), there should at least be a requirement in the by-laws that says that the method will be chosen **and publicly declared** prior to the election. 

(In the past we have had requests to change the specific counting methodology after an election has started, which I don't think is fair. By making it a requirement to select and declare the methodology prior to the election starting it means we can keep it fair for the current election, while maintaining the flexibility to change it to other methods in subsequent elections.)

Reply:
I agree with @ahankinson that there should be a requirement that election rules, including counting methodology, be chosen and publicly declared before the election. 

I may be the source of one of the requests in recent years to change the specific counting method after an election had started. If I am recalling the incident correctly, I made my request as soon as I found out about the election's counting method, but I only found out after the election had started. Thus, to do any good, the public declaration needs to be in advance of the election starting. The announcement needs to be early enough that the membership and potential candidates can hear the announcement, make suggestions, and get the suggestions acted on, all before the election starts.

My suggestion is that the Board treat "prepare for election" as a separate task, 3-6 months in advance of the election. That task includes choosing the Election Officers, calling for comment on the election rules, and acting on comments received. 

A wise Board might also start the process of proactively recruiting new Board candidates at the same time. Not all good Board candidates volunteer without persuasion. But that is outside the scope of this By-Law.

Reply:
Thanks for the suggestion. We included a statement about the public declaration of the election procedure before the nomination phase. Further explanations and recommendations will go into the best practice document for the election officers.